api/src/test/java/io/opentelemetry/common/ImmutableAttributesTest.java

```diff
@@ -31,7 +31,7 @@
 import org.junit.rules.ExpectedException;
 
 /** Unit tests for {@link Attributes}s. */
-public class AttributesTest {
+public class ImmutableAttributesTest {
```

Comment:
This should stay as `AttributesTest`, no?

Reply:
Yep, will fix.

Reply:
fixed